code/DataObjects/Slice.php

```diff
@@ -36,9 +37,10 @@ class Slice extends DataObject
     );
 
     private static $has_one = array(
-        'Parent' => SiteTree::class
+        'Parent' => GenericContentPage::class
```

Comment:
hey Baptist,

GenericContentPage is not part of core SilverStripe. This is a class specific to Caltex site so cannot be used in this module.